lib/ReactViews/FeatureInfo/FeatureInfoSection.jsx

```diff
@@ -89,6 +91,9 @@ const FeatureInfoSection = createReactClass({
                     longitude: CesiumMath.toDegrees(latLngInRadians.longitude)
                 };
             }
+            if (this.props.clock.currentTime) {
```

Comment:
Check that `this.props.clock` is defined, too.  In at least one case (when the user clicks on a location search result marker), it won't be.